scripts/build.ts

```diff
@@ -3,6 +3,21 @@ async function main() {
     cmd: ["wasm-pack", "build", "--target", "web", "--release"],
   }).status();
   await Deno.remove("./pkg/.gitignore");
+  const wasmFile = JSON.parse(
```

Comment:
Rather than using `Deno.readTextFile` + `JSON.parse`, you can use a dynamic import: https://docs.deno.com/examples/importing-json/.